### DISCUSSION TITLE
Roadmap for version 1.0 release of Microsoft.Build.Sql

### DISCUSSION THREAD

**Discussion by IanKemp** [microsoft/DacFx#188] [General]
Is there one? Even a intended quarter would be helpful.

**Comment by ChristopherHaws** [comment]
@IanKemp There is a [project which shows the items](https://github.com/microsoft/DacFx/projects/1) and their states. Not sure if there is any other info; if there is, I couldn't find it. Really looking forward to this SDK so I can get full tooling support back (I am currently using [MSBuild.Sdk.SqlProj](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj)).

**Comment by czb182** [comment]
I wonder the same.  When it was introduced 2 years ago I was excited, never would have thought 2 years later would still be in preview status.  Hopefully Microsoft can dedicate the resources to get this out the door.  Will be 2024 soon and might still be having legacy project formats in our solution

**Comment by amselem** [comment]
Any news?

**Comment by tscamell** [comment]
To quote the OP: "Is there one? Even an intended ~~quarter~~ year would be helpful"

To not have to use .NET Framework for our sql project builds would be really something. I find it hard to commit our team to using a preview package, when it might still be preview (or worse, dropped) in the next 3 years.

Any news on plans/blockers for the main release would be greatly appreciated :)

**Reply by ErikEJ** [reply]
@tscamell In the meantime you can use MSBuild.SDK.SqlProj (I am a maintainer)
